README.md

```diff
@@ -137,36 +137,51 @@ pip install -e . # pip install -e .[all] to install all optional extras (see "In
 
 ## <a name="SparkSetup"></a>Local Spark Setup
 
-If you're running Spark locally, you'll need to add the rikai jar when creating the Spark session.
-If you want to read/write data from/to S3, you'll need to add additional options as well.
-
-Add appropriate options when creating the SparkSession:
+If you're running Spark locally, you'll need to add the rikai jar when creating the Spark session:
 
 ```python
 spark = (
    SparkSession
       .builder
-      .appName('rikai')
+      .appName('rikai-quickstart')
       .config('spark.jars.packages', 'ai.eto:rikai:0.0.1')
-      .config("spark.driver.extraJavaOptions", "-Dcom.amazonaws.services.s3.enableV4=true")
-      .config('spark.jars.packages', 'org.apache.hadoop:hadoop-aws:2.7.4')
-      # ... other options
       .master("local[*]")
       .getOrCreate()
 )
 ```
 
-Please note that the above sample assumes your local Apache Spark comes with Hadoop 2.7. If you
-installed another version of Hadoop, please use a matching hadoop-aws jar version.
-
-As with other Spark options, there are multiple ways to specify them.
-Please see [Spark documentation](https://spark.apache.org/docs/latest/configuration.html) for
-details.
+If you want to read/write data from/to S3, you will need additional setup:
+
```

Comment:
feel that it is too crowd in `README.md` tho. should we have a ``docs/`` section just for AWS/S3 setup?

No urgent.

Reply:
yeah good idea. It started out simple but really snowballed. I'll move it to docs/ section, along with the databricks stuff.